### ISSUE TITLE
Problem with docker run command

### ISSUE THREAD

**Issue by sivianil** [google/deepvariant#593]
Hey, I tried to run the docker run single command following step by step from your instructions on macOS. Unfortunately, I ran into an error that couldn't give me any idea what's wrong! I attached the screenshot, so you can see the error. 
<img width="1512" alt="Screenshot 2022-12-01 at 14 28 42" src="https://user-images.githubusercontent.com/75676816/205064985-34617012-9e82-4734-88a6-0d702f7bd445.png">

Thanks,
Anil

**Comment by danielecook** [comment]
@sivianil it looks to me like an issue with the location of your data. The error is saying that the reference file does not exist.

Can you double check the following:

1. Did you define `${INPUT_DIR}`?
2. Do you have the `ucsc.hg19.chr20.unittest.fasta` in the input directory?

**Comment by sivianil** [comment]
Yeah I defined INPUT_DIR and I have all the required files in my INPUT_DIR. 
<img width="1011" alt="Screenshot 2022-12-01 at 15 16 55" src="https://user-images.githubusercontent.com/75676816/205075692-189e08af-4ea8-44b7-ab1a-c812f170fa8e.png">

**Comment by danielecook** [comment]
Try running the image interactively and verifying that the mounting works as expected.

```bash
sudo docker run \
  -v "${INPUT_DIR}":"/input" \
  -v "${OUTPUT_DIR}":"/output" \
  google/deepvariant:"${BIN_VERSION}" \
  /bin/bash
```

If you run the command this way you should be able to type:

```bash
ls /input
```

and see the `${INPUT_DIR}` listing of files. You can also run the command directly from there:

```bash
/opt/deepvariant/bin/run_deepvariant ...
```
